nrf_wifi/utils/src/util.c

```diff
@@ -128,12 +129,135 @@ int nrf_wifi_utils_chan_to_freq(struct nrf_wifi_osal_priv *opriv,
 		break;
 	default:
 		nrf_wifi_osal_log_err(opriv,
-				      "%s: Invalid band value %d",
-				      __func__,
-				      band);
+					  "%s: Invalid band value %d",
+					  __func__,
+					  band);
 		goto out;
 	}
 out:
 	return freq;
 
 }
+
+void nrf_wifi_init_ph_rf_params(struct nrf_wifi_phy_rf_params *phy_rf_params,
+								unsigned char package_info)
+{
+	/* Initilaize reserved bytes*/
+
+	memset(&phy_rf_params->reserved, 0, sizeof(phy_rf_params->reserved));
+
+	/*Initialize PD adjust values for MCS7. Currently these 4 bytes are not being used*/
+	phy_rf_params->pd_adjust_val.pd_adjt_lb_chan = PD_ADJUST_VAL;
```

Comment:
Either shorten `phy_rf_params` or take intemediate pointers to fix indentation.

Reply:
Done